app/javascript/packs/MyHeader.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types'
+import AddNewRestPopup from './AddNewRestPopup'
+
+class MyHeader extends React.Component {
+
+    render() {
+        return (
+            <div className="my_header">
+                <div className="header_text">
+                    <div className="main_title">WeEat!</div>
```

Comment:
`h1`

Reply:
I tried not to use h1 and other html styling cause I thought the better way is to do it using css.. isn't it?..

Reply:
You can use both, use H1 and style it in CSS, it's just important to use the right html semantics.

Reply:
ok. Changed, thanks!